.travis.yml

```diff
@@ -4,5 +4,5 @@ mono:
   - latest
   - 4.6.2
 script: 
-  - ./build.sh -target="Travis" -configuration="Release" -verbosity="verbose"
-    
+  - git fetch --unshallow
+  - ./build.sh --target "Travis"
```

Comment:
I've pushed a commit b758c79 to correct the error from the merge (the error made Travis fail every time).